platform-dom/src/main/resources/com/softicar/platform/dom/dom-image-viewer-style.css

```diff
@@ -1,16 +1,37 @@
 .DomImageViewer {
 	align-items: center;
+	display: grid;
+	grid-template-rows: min-content auto;
+	min-width: min-content;
+	overflow: hidden;
+	resize: both;
 }
 
-.DomImageViewer.draggable {
-	height: 85vh;
-	overflow: scroll;
+.DomImageViewerCanvas {
+	align-self: stretch;
+	min-height: 50px;
+	overflow: auto;
+	position: relative;
+	width: 100%;
 }
 
 .DomImageViewerImage {
-	box-shadow: 0px 0px 3px #777777;
+	box-shadow: 0px 0px 3px 3px #ccc;
 }
 
-.DomImageViewerImage.rotated {
-	transform: rotate(180deg);
+.DomImageViewerImageHolder {
+	box-sizing: border-box;
+	position: relative;
+	padding: 10px;
+}
+
+.DomImageViewerPageIndicator {
+	white-space: nowrap;
+}
+
+.DomImageViewerToolBar {
```

Comment:
For the sake of alignment (red lines), it might look better if that toolbar had horizontal paddings that match those of the displayed document:
![image](https://user-images.githubusercontent.com/15086658/188572513-6fc9a96f-78d2-49d9-9d2f-f17fe99e3f96.png)

Reply:
I kind of grew to dislike the page-turning button icons. Those look too subtle, esp. when compared to the zoom icons... tbh, they don't really look like something that can be clicked.
I think we should use one of the following pairs instead, from our usual icon set:
![image](https://user-images.githubusercontent.com/15086658/188573178-9c6732d9-7b3b-462b-bd03-98c0e332dfea.png)
![image](https://user-images.githubusercontent.com/15086658/188573331-23bbcfd7-e644-4687-bad5-6b7e72755c23.png)

Reply:
done